### PR TITLE
WebExplore redirect after purchase

### PR DESCRIPTION
**Pull request by valentinamackovic**
### References Issues:

[Asana](https://app.asana.com/0/1141185905028320/1168967382452721/f)

### Description:

Added redirecting the user to My Tickets tab after a successful purchase in WebExplore.

### Release Screenshots / Video:

[Preview](https://share.getcloudapp.com/yAuv5G1n)

### Environment Variables:
 * No change

### CODE REVIEW

File: src/WebExplore/Explore.tsx [hunk 1]
```diff
@@ -61,15 +61,10 @@ class Explore extends React.Component<ExploreProps, any> {
...
-    let pathname = '/'
-    try {
-      const urlData = JSON.parse(data)
-      pathname = urlData.pathname || '/'
-    } catch (e) {}
-    if (pathname !== '/') {
-      navigation.setParams({ hasBack: '1' })
-    } else {
-      navigation.setParams({ hasBack: '0' })
+    const urlData = JSON.parse(data)
```
[the remark this answers is not in the document]
Comment (Krakaw) [reply src/WebExplore/Explore.tsx:64]: The new type is in here: https://github.com/big-neon/bn-mobile-react/pull/350

File: src/WebExplore/Explore.tsx [hunk 2]
```diff
@@ -61,19 +61,15 @@ class Explore extends React.Component<ExploreProps, any> {
     const { navigation } = this.props
     const { nativeEvent = {} } = event
     const { data = '{}' } = nativeEvent
-    let pathname = '/'
     try {
       const eventMessage = JSON.parse(data)
       const urlData = eventMessage.json
       if (eventMessage.type === 'navigation') {
-        pathname = urlData.pathname || '/'
+        if ( urlData.search ==='?purchase_successful=true') {
```
Comment (Krakaw) [inline src/WebExplore/Explore.tsx:68]: What happens if there are utm parameters or some other garbage? Would doing something like this work?
```suggestion
        if ( urlData.search.indexOf('purchase_successful=true') > -1) {
```

Comment (Krakaw) [reply src/WebExplore/Explore.tsx:68]: This also future proofs in case we want to pass more props back through the url in the future.

Comment (valentinamackovic) [reply src/WebExplore/Explore.tsx]: I used includes because it returns a boolean value.

**Pull request merged**